toc/ROLES.md

```diff
@@ -49,6 +49,20 @@ establish sub-working groups. Working groups delegate change approval to Approve
     </tr>
   </thead>
 
+  <tr>
+    <td>(everyone)</td>
```

Comment:
I like the idea of pointing out this more casual tier of participation, but it might benefit from having a distinct name. Maybe "Contributor" or "Participant"?

Reply:
Also, as @stephanme mentioned, it should have a corresponding section below this table in this document.

Reply:
Ok, I think this is clearer now that we've separated out a casual contributor tier from this.